rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/RocketChatWidgets.java

```diff
@@ -0,0 +1,26 @@
+package chat.rocket.android.widget;
+
+import android.content.Context;
+import com.facebook.drawee.backends.pipeline.DraweeConfig;
+import com.facebook.drawee.backends.pipeline.Fresco;
+import com.facebook.imagepipeline.backends.okhttp3.OkHttpImagePipelineConfigFactory;
+import com.facebook.imagepipeline.core.ImagePipelineConfig;
+
+import chat.rocket.android.widget.fresco.ImageFormatConfigurator;
+import okhttp3.OkHttpClient;
+
+public class RocketChatWidgets {
```

Comment:
From an external caller (`app` module) what we are initializing is the `RocketChatWidgets` module, not a specific library inside it.